src/config.js

```diff
@@ -40,22 +51,10 @@ if (isDev) {
   }))
 }
 
-// Default settings for new windows
-const window = {
-  icon: getLogo(),
-  title: 'IPFS Dashboard',
-  autoHideMenuBar: true,
-  width: 800,
-  height: 500,
-  webPreferences: {
-    webSecurity: false
-  }
-}
-
 // Configuration for the MenuBar
 const menubar = {
   dir: __dirname,
-  width: 300,
+  width: 850,
   height: 400,
   index: `file://${__dirname}/views/menubar.html`,
   icon: (os.platform() === 'darwin') ? macOsMenuBar : getLogo(),
```

Comment:
Can you change this icon on the fly? I believe that you can.

I see a dark gray icon because that is the default for MacOS, but before it was changing to the blue one when the node got started.

Reply:
@hacdias this is the culprit for the dark gray IPFS logo on my machine.

Reply:
Did you try again after I removed the context menu? Cause I think it might be related!

Reply:
Yes -- https://github.com/ipfs-shipyard/station/pull/516#issuecomment-349065402 --, still gray though

Reply:
Strange... I don't know what I might have changed related to that... And I can't really try that out 😮 Could you check that, please?

Reply:
@hacdias you can try it indeed. You also should experience on Windows

- Daemon off - Gray logo
- Daemon on - Blue logo

Reply:
Done on 232ba2d